src/mood.ts

```diff
@@ -24,12 +27,27 @@ import {
   use,
   useSkill,
 } from "kolmafia";
-import { getActiveSongs, have, isSong } from "./lib.js";
+import { getActiveSongs, have, isSong, unequip } from "./lib.js";
 import { get } from "./property.js";
 import { AsdonMartin } from "./resources/index.js";
-import { $item, $skill } from "./template-string.js";
+import { $effect, $item, $skill, $slot } from "./template-string.js";
 import { clamp, sum } from "./utils.js";
 
+const shieldEffects = new Map<Effect, { skill: Skill; turnsPerCast: number }>([
+  [
+    $effect`Thoughtful Empathy`,
+    { skill: $skill`Empathy of the Newt`, turnsPerCast: 5 },
+  ],
+  [
+    $effect`Lubricating Sauce`,
+    { skill: $skill`Sauce Contemplation`, turnsPerCast: 5 },
```

Comment:
I misremembered these as giving fewer adventures than the regular spell does; looks like we _can_ rely on turnsPerCast after all, just as sort-of-a-coincidence. Sorry king